tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_reach_push_pick_place.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
```

Comment:
your tests can import `metaworld.policies.ScriptedReachPushPickPlacePolicy` and verify it solves the environment.